framework/resources/Datasources/ctas/create_pushDownData.ddl

```diff
@@ -0,0 +1,13 @@
+alter session set `store.format` = 'parquet';
```

Comment:
For alter session, please just use 'set' instead of 'alter session set'. One, that's more readable and two, it's what we've been following in this repo.

Reply:
also parquet is the default format - any reason to explicitly set this?